<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonsDialog</class>
 <widget class="QDialog" name="PersonsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="inputPName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPGender">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPBirthYear">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputPDeathYear">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGender">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBirthYear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BirthYear:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDeathYear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DeathYear:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;American Typewriter&quot;;
font: 18pt &quot;.SF NS Text&quot;;</string>
   </property>
   <property name="text">
    <string>Persons</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddPerson">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add or edit person</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
